uv.lock

```diff
@@ -0,0 +1,745 @@
+version = 1
```

Comment:
sphinx is a library, not an application, it doesn't make sense for it to have a committed lock file

Reply:
That advice is a little outdated. These days the recommendation is generally to commit the lockfiles for libraries too if and only if you use an automated tool for lockfile maintenance.

The lockfiles gives you deterministic CI builds and ensures contributors use a consistent environment.

See https://blog.rust-lang.org/2023/08/29/committing-lockfiles.html (this is a blog post about cargo, but it applies here too)